PROBot/BotClient.cs

```diff
@@ -176,6 +176,11 @@ public void Update()
                 return;
             }
 
+            if (Game != null && Game.IsInBattle && Game.IsInactive && AI != null && AI.UseMandatoryAction())

```

Comment:
Good catch, but this probably belongs to PROProtocol if the player cannot influence it.
PROProtocol should basically be enough to implement a full game client (without any cheat).
PROBot is just calling some PROProtocol methods to simulate the player movements, as if we were clicking.

Reply:
I don't know where else I can put this while still by-passing the 'bot started' check.